terapia/models.py

```diff
@@ -651,16 +650,26 @@ def get_hora_por_indice(indice):
             timedelta_hora = indice * CONSULTA_DURACAO
             return time(timedelta_hora.seconds // 3600, (timedelta_hora.seconds // 60) % 60)
 
-        def segunda_a_domingo(matriz_disponibilidade_booleanos):
-            matriz_disponibilidade_booleanos.append(matriz_disponibilidade_booleanos.pop(0))
+        def segunda_a_domingo(matriz):
+            matriz.append(matriz.pop(0))
 
         def to_dia_semana_iso(indice):
             return indice % 7 + 1
 
-        disponibilidade = []
-        m = json.loads(matriz_disponibilidade_booleanos)
+        if isinstance(matriz_disponibilidade_booleanos, (str, bytes, bytearray)):
+            m = json.loads(matriz_disponibilidade_booleanos)
+        else:
+            m = matriz_disponibilidade_booleanos
```

Comment:
Esse if foi colocado aqui por quê? Tinha algum trecho do código enviando a matriz na forma de lista em vez de string? Se sim, essa parte do código tem que ser ajeitada, porque essa função _sempre_ deve receber a matriz na forma de string pra que possa ser convertida usando json.loads()

```suggestion
        m = json.loads(matriz_disponibilidade_booleanos)
```